src/program/spoon/README.md

```diff
@@ -0,0 +1,41 @@
+## Inter-process sample program
+
+The `program/spoon` directory contains a sample (and test, and benchmark)
+for the inter_proc app.
```

Comment:
Include explanation of the name "spoon"? (I suspect there is a pun that I am missing completely?)